updatecli/updatecli.d/launchable.yml

```diff
@@ -14,23 +14,18 @@ scms:
       branch: "{{ .github.branch }}"
 
 sources:
-  lastReleaseVersion:
-    kind: githubrelease
-    name: Get the latest launchable release version
+  currentPackerImageVersion:
+    kind: file
+    name: Get the launchable version from jenkins-infra/packer-images
     spec:
-      owner: "launchableinc"
-      repository: "cli"
-      token: "{{ requiredEnv .github.token }}"
-      username: "{{ .github.username }}"
-      versionfilter:
-        kind: semver
-    transformers:
-      - trimprefix: v
+      file: https://github.com/jenkins-infra/packer-images/blob/main/provisioning/tools-versions.yml
```

Comment:
We must use the latest published release, like with Maven in https://github.com/jenkins-infra/jenkins-infra/blob/34d88cb1a44143dc352f4e3869528e019107cf20/updatecli/weekly.d/jenkinscontroller-tools-maven.yaml#L28-L31 

The reason is to avoid a discrepancy between the reality on ci.jenkins.io (e.g. the released version) and the PR in this repository.

=> this is a blocker (because we had too many problem with the Maven version discrepancies in the past)